Cargo.lock

```diff
@@ -3286,9 +3273,9 @@ checksum = "cda74da7e1a664f795bb1f8a87ec406fb89a02522cf6e50620d016add6dbbf5c"
 
 [[package]]
 name = "tokio"
-version = "1.14.0"
+version = "1.12.0"
```

Comment:
We should probably not downgrade tokio. Especially since `1.12.0` has a CVE reported to it. Not sure if this downgrade happened by accident or what caused it?